model-client/src/commonMain/kotlin/org/modelix/model/client2/ModelClientV2.kt

```diff
@@ -192,12 +197,12 @@ class ModelClientV2(
             url {
                 takeFrom(baseUrl)
                 appendPathSegmentsEncodingSlash("repositories", branch.repositoryId.id, "branches", branch.branchName)
+                parameters["format"] = "plain"
```

Comment:
Some constant / enum for "plain" etc.?

Reply:
Extracted it into a new function to remove the duplication of the string literals.